<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetFiducialMarkDialog</class>
 <widget class="QDialog" name="SetFiducialMarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Fiducial Mark</string>
  </property>
  <widget class="QGroupBox" name="groupBoxStandardShape">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelShape">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxShape">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSize">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSizeUnit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDark">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark Body</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbUseRealImage">
      <property name="text">
       <string>Use Real Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbUseStandardShape">
      <property name="text">
       <string>Use Standard Shape</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetFiducialMarkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetFiducialMarkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
